### ISSUE TITLE
formats config doesn't work as global config

### ISSUE THREAD

**Issue by thierrybellevue** [KillerCodeMonkey/ngx-quill#332]
The formats config doesn't seem to work as a global config. 

I tried:
```
    QuillModule.forRoot({
      formats: ['bold', 'italic', 'list', 'header']
    })
```
and it doesn't work.

But when I try: 
`<quill-editor [formats]="['bold', 'italic', 'list', 'header']"></quill-editor>` 
it works

**Comment by KillerCodeMonkey** [comment]
thanks for noticing! i will look into it.

**Comment by KillerCodeMonkey** [comment]
should be fixed in 4.6.5

**Comment by KillerCodeMonkey** [comment]
check out 4.6.9 :)
